wallet-new/test/unit/Test/Spec/CoinSelection.hs

```diff
@@ -64,21 +64,37 @@ import           Test.Spec.CoinSelection.Generators (InitialBalance (..),
 freeLunch :: Int -> NonEmpty Core.Coin -> Core.Coin
 freeLunch _ _ = Core.mkCoin 0
 
+freeLunchCheck :: Core.Coin -> Bool
+freeLunchCheck c = c == Core.mkCoin 0
+
 -- | The smallest fee possible.
 minFee :: Int -> NonEmpty Core.Coin -> Core.Coin
 minFee _ _ = Core.mkCoin 1
 
+minFeeCheck :: Core.Coin -> Bool
+minFeeCheck c = c == Core.mkCoin 0
```

Comment:
typo ? Shouldn't this be `c == Core.mkCoin 1` ?

Reply:
yes, strange..